configuration/restdocs.md

```diff
@@ -7,14 +7,29 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
+The REST API gives the ability to access a wide range of information associated with a running openHAB installation, and the ability to invoke changes to the openHAB installation. 
+The REST API provides an interface that may only be accessed programmatically.
+The information available from the API includes Items and their State, installed bindings, sitemap information, and much more. 
```

Comment:
state lowercase

Reply:
So wrapping the information in these lines along with the important information about server-push, how about:

The information available from the API includes Items and their state, installed bindings, sitemap information, and much more. The information exchange takes place through HTTP and the API provides a standardized platform for information exchange among different computer systems.  The REST API also supports server-push, so you can subscribe to certain resources and receive change notification for those resources when they occur.

Reply:
changed